contrib/tracefile-analyser.php

```diff
@@ -182,7 +211,7 @@ private function parseLine( $line )
 
 				// collapse data onto functions array
 				$dTime   = $time   - $prevTime;
-				$dMemory = $memory - $prevMem;
+				$dMemory = (int) $memory - (int) $prevMem;
```

Comment:
That shouldn't be necessary, as those values should always be integers already. How did this throw an error?